installer.sh

```diff
@@ -0,0 +1,442 @@
+
+set -euo pipefail
+
+# Location of install log
+: "${INSTALL_LOG:=/tmp/platformsh-install-$(date '+%Y%m%d-%H%M%S')}.log}"
+
+# Define this to force install method
+: "${INSTALL_METHOD:=}"
+
+: "${URL:=https://github.com/platformsh/cli/releases/download}"
+
+# Force Platform.sh CLI installation in this directory instead of system directory
+: "${INSTALL_DIR:=}"
+
+# Platform.sh CLI version to install
+: "${VERSION:=}"
+
+# macOS specifics
+: "${BREW_TAP:=platformsh/tap}"
+: "${BREW_FORMULA:=platformsh/tap/platformsh-cli}"
+
+# GitHub token check
+: "${GITHUB_TOKEN:=}"
+
+# global variables
+binary="platform"
+cmd_shasum=""
+cmd_sudo=""
+dir_bin="/usr/bin"
+footer_notes=""
+has_sudo=""
+kernel=""
+machine=""
+version=""
+
+# create a log file where every output will be pipe to
+pipe=/tmp/platformsh-install-$$.tmp
+mkfifo $pipe
+tee < $pipe $INSTALL_LOG &
+exec 1>&-
+exec 1>$pipe 2>&1
+trap 'rm -f $pipe' EXIT
+
+function output {
+    style_start=""
+    style_end=""
+    if [ "${2:-}" != "" ]; then
+    case $2 in
+        "success")
+            style_start="\033[0;32m"
+            style_end="\033[0m"
+            ;;
+        "error")
+            style_start="\033[31;31m"
+            style_end="\033[0m"
+            ;;
+        "info"|"warning")
+            style_start="\033[33m"
+            style_end="\033[39m"
+            ;;
+        "heading")
+            style_start="\033[1;33m"
+            style_end="\033[22;39m"
+            ;;
+        "comment")
+            style_start="\033[2m"
+            style_end="\033[22;39m"
+            ;;
+    esac
+    fi
+
+    builtin echo -e "${style_start}${1}${style_end}"
+}
+
+function exit_with_error() {
+    output "+--------------------------------------------------+" "error"
+    output "|                                                  |" "error"
+    output "|              Installation failed                 |" "error"
+    output "|                                                  |" "error"
+    output "+--------------------------------------------------+" "error"
+
+    output "\nGet help with your Platform.sh installation:" "heading"
+    output "  Inspect the logs: ${INSTALL_LOG}"
+    output "  Website:  https://platform.sh"
+
+    exit 1
+}
+
+function intro() {
+    output "+--------------------------------------------------+" "heading"
+    output "|                                                  |" "heading"
+    output "|             Platform.sh Installer                |" "heading"
+    output "|                                                  |" "heading"
+    output "+--------------------------------------------------+" "heading"
+
+    output "\nChecking Environment" "heading"
+}
+
+function outro() {
+    output ""
+    output "+--------------------------------------------------+" "success"
+    output "|                                                  |" "success"
+    output "|   Platform.sh has been installed successfully.   |" "success"
```

Comment:
this looks odd to me because I know "Platform.sh" is a lot more than a single binary

Reply:
Fixed the wording: https://github.com/platformsh/cli/pull/38/files#diff-a66c904fc30426ab62d77441e63aaadecc1b1beaea08f08321681d8f03301187R102-R108